Source/ARTTypes.h

```diff
@@ -221,88 +403,129 @@ NSString *generateNonce(void);
 
 @end
 
-#pragma mark - ARTChannelOccupancy
-
+/**
+ * BEGIN CANONICAL PROCESSED DOCSTRING
+ * Contains the metrics of a `ARTRestChannel` or `ARTRealtimeChannel` object.
+ * END CANONICAL PROCESSED DOCSTRING
+ */
 @interface ARTChannelOccupancy : NSObject
 
+/**
+ * BEGIN CANONICAL PROCESSED DOCSTRING
+ * A `ARTChannelMetrics` object.
+ * END CANONICAL PROCESSED DOCSTRING
+ */
 @property (nonatomic, strong, readonly) ARTChannelMetrics *metrics;
 
 - (instancetype)initWithMetrics:(ARTChannelMetrics *)metrics;
 
 @end
 
-#pragma mark - ARTChannelStatus
-
+/**
+ * BEGIN CANONICAL PROCESSED DOCSTRING
+ * Contains the status of a `ARTRestChannel` or `ARTRealtimeChannel` object such as whether it is active and its `ARTChannelOccupancy`.
+ * END CANONICAL PROCESSED DOCSTRING
+ */
 @interface ARTChannelStatus : NSObject
 
+/**
+ * BEGIN CANONICAL PROCESSED DOCSTRING
+ * If `true`, the channel is active, otherwise `false`.
+ * END CANONICAL PROCESSED DOCSTRING
+ */
 @property (nonatomic, readonly) BOOL active;
 
+/**
+ * BEGIN CANONICAL PROCESSED DOCSTRING
+ * A `ARTChannelOccupancy` object.
+ * END CANONICAL PROCESSED DOCSTRING
+ */
 @property (nonatomic, strong, readonly) ARTChannelOccupancy *occupancy;
 
 - (instancetype)initWithOccupancy:(ARTChannelOccupancy *)occupancy active:(BOOL)active;
 
 @end
 
-#pragma mark - ARTChannelDetails
-
+/**
+ * BEGIN CANONICAL PROCESSED DOCSTRING
+ * Contains the details of a `ARTRestChannel` or `ARTRealtimeChannel` object such as its ID and `ARTChannelStatus`.
+ * END CANONICAL PROCESSED DOCSTRING
+ */
 @interface ARTChannelDetails : NSObject
 
+/**
+ * BEGIN CANONICAL PROCESSED DOCSTRING
+ * The identifier of the channel.
+ * END CANONICAL PROCESSED DOCSTRING
+ */
 @property (nonatomic, strong, readonly) NSString *channelId;
 
+/**
+ * BEGIN CANONICAL PROCESSED DOCSTRING
+ * A `ARTChannelStatus` object.
```

Comment:
This needs a bit more explanation

Reply:
I think this should also be raised at canonical repo.

Reply:
https://github.com/ably/canonical-api-reference-prototyping/issues/25#issue-1367687965